### PR TITLE
Templates bundle yaml file

### PR DESCRIPTION
**Pull request by saltiyazan**
Uses jinja template to allow taking the channel of the charms as a parameter

# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: orchestrator-bundle/orc8r-bundle/render_bundle.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""This script renders the jinja template given to it.
+
+This scripts takes a jinja template of the magma-orc8r bundle,
+and renders the bundle into an output file after adding the value of the cannel.
```
Comment (gruyaume) [inline orchestrator-bundle/orc8r-bundle/render_bundle.py]:
```suggestion
and renders the bundle into an output file after adding the value of the channel.
```

File: orchestrator-bundle/orc8r-bundle/render_bundle.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        help="bundle template to render",
+        default=BUNDLE_TEMPLATE_NAME,
+    )
+    parser.add_argument(
+        "--output",
+        type=Path,
+        help="path to the rendered bundle yaml",
+        required= True
+    )
+    parser.add_argument(
```
Comment (gruyaume) [inline orchestrator-bundle/orc8r-bundle/render_bundle.py:34]: We could limit the channel options to whatever is supported by charmhub

File: orchestrator-bundle/orc8r-bundle/render_bundle.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    )
+    bundle_args, _= parser.parse_known_args()
+
+    return bundle_args.template, bundle_args.output, bundle_args.channel
+
+if __name__ == "__main__":
+    template, output, channel = parse_args()
+    env = jinja2.Environment(loader=jinja2.FileSystemLoader("./"))
+    t = env.get_template(template)
+    with open(template) as t:
```
Comment (gruyaume) [inline orchestrator-bundle/orc8r-bundle/render_bundle.py:48]: It's a bit confusing that we reuse `t` that's already defined a line earlier. Plus `t` is not the greatest variable name

File: orchestrator-bundle/orc8r-bundle/render_bundle.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```
Comment (gruyaume) [inline orchestrator-bundle/orc8r-bundle/render_bundle.py:1]: (Not related to this file)
We should update the CI to render the bundle before publishing it.

File: orchestrator-bundle/orc8r-bundle/render_bundle.py (hunk 1 above)
Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/render_bundle.py]: Fixed

File: orchestrator-bundle/orc8r-bundle/render_bundle.py (hunk 2 above)
Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/render_bundle.py:34]: Fixed

File: orchestrator-bundle/orc8r-bundle/render_bundle.py (hunk 3 above)
Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/render_bundle.py:48]: Actually `t = env.get_template(template)` is not at all needed, it remained there as I was testing the script and forgot to remove it. it's fixed now. Thanks

File: orchestrator-bundle/orc8r-bundle/render_bundle.py (hunk 4 above)
Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/render_bundle.py:1]: `orchestrator.bundle.upload.edge.yml` is updated now accordingly.

**Review by gruyaume — APPROVED** [review]

**Pull request merged**